lua/weapons/weapon_ace_portablemortar/cl_init.lua

```diff
@@ -0,0 +1,241 @@
+include("shared.lua")
+
+function SWEP:DrawScope(Zoomed)
+	if not (Zoomed and self.HasScope) then return end
+
+	local scrw = ScrW()
+	local scrh = ScrH()
+
+	surface.SetDrawColor(0, 0, 0, 255)
+
+	local rectsides = ((scrw - scrh) / 2) * 0.7
+
+	surface.SetMaterial(Material("gmod/scope"))
+	surface.DrawTexturedRect(rectsides, 0, scrw - rectsides * 2, scrh)
+	surface.DrawRect(0, 0, rectsides + 2, scrh)
+	surface.DrawRect(scrw - rectsides - 2, 0, rectsides + 2, scrh)
+end
+
+
+function SWEP:DoDrawCrosshair(x, y)
+	local Zoom = self:GetZoomState()
+	self:DrawScope(Zoom)
+
+	local owner = self:GetOwner()
+	local inaccuracy = math.min(owner:GetVelocity():Length() / owner:GetRunSpeed(), 1)
+	inaccuracy = math.max(inaccuracy, self.Heat / self.HeatMax)
+--	local fusedelay = self:GetFuseDelay()
+	local Tx = self:GetPx()
+	local ST = self:GetPy()
+	local Tg = self:GetPg()
+
+	if Zoom then
+
+		local RT = util.QuickTrace(owner:GetShootPos(), owner:GetAimVector() * 50000, {owner})
+
+		local RDist = 9999
+
+		if RT.Hit then
+			local difpos = RT.HitPos - owner:GetShootPos()
+			RDist = ( difpos * Vector(1,1,0) ):Length() / 39.37
+		end
+
+
+--			surface.SetDrawColor(Color(255, 0, 0, 255 - inaccuracy * 150))
+
+	if ST == 2 then
+		surface.SetDrawColor(Color(150, 150, 150, 255))
+	else
+		surface.SetDrawColor(Color(150, 30, 30, 255))
+	end
```

Comment:
```suggestion
		if ST == 2 then
			surface.SetDrawColor(Color(150, 150, 150, 255))
		else
			surface.SetDrawColor(Color(150, 30, 30, 255))
		end
```

more funny indenting